pkgs/development/python-modules/livelossplot/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, matplotlib, notebook }:
+
+buildPythonPackage rec {
+  pname = "livelossplot";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner  = "stared";
+    repo   = pname;
+    rev    = "v.${version}";
+    sha256 = "0hih8z6y8pb1khp9nf8r4v4a4lv0rzrisd350qiakim83kakfy10";
+  };
+
+  propagatedBuildInputs = [ matplotlib notebook ];
+
+  doCheck = false;
```

just to give some assurance that the package is actually usable.
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "livelossplot" ];
```

---

Done.